cmake/cpack_settings.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# CPack package handling
+include(InstallRequiredSystemLibraries)
+set(CPACK_PACKAGE_NAME "open-enclave")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Open Enclave SDK")
+set(CPACK_PACKAGE_CONTACT "openenclave@microsoft.com")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
+set(CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/LICENSE")
+set(CPACK_PACKAGE_VERSION ${OE_VERSION})
+set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "libsgx-enclave-common (>=2.3.100.46354-1), libsgx-enclave-common-dev (>=2.3.100.0-1), libsgx-dcap-ql (>=1.0.100.46460-1.0), libsgx-dcap-ql-dev (>=1.0.100.46460-1.0), pkg-config")
+
+# Cpack variables for quote verification component
+set(CPACK_DEBIAN_OEQUOTE_PACKAGE_NAME "open-enclave-quote")
+set(CPACK_DEBIAN_OEQUOTE_FILE_NAME DEB-DEFAULT)
+set(CPACK_COMPONENT_OEQUOTE_DESCRIPTION "Open Enclave Quote Verification Library")
+set(CPACK_DEBIAN_OEQUOTE_PACKAGE_DEPENDS "pkg-config")
+include(CPack)
```

Comment:
As you mentioned to me yesterday, it might be a good idea to make these packages conflict with each other! Looks like cpack has some variables you can set to do this: `CPACK_DEBIAN_PACKAGE_CONFLICTS`

https://cmake.org/cmake/help/v3.13/cpack_gen/deb.html#cpack_gen:CPack%20DEB%20Generator

Reply:
It's not obvious to me that making them mutually exclusive is the best solution here. Are we saying that we don't think devs would want to author both the normal-mode-only challenger as well as an enclave app on the same dev box?

Reply:
That's a really good point that I didn't think of, Simon. I agree, these should not conflict with each other